setup.cfg

```diff
@@ -14,7 +14,8 @@ classifier =
     Environment :: Other Environment
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
-    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
```

Comment:
This needs to be changed when publishing for 3.6?

Reply:
yes, it will